airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeDestination.java

```diff
@@ -42,7 +42,10 @@ public SerializedAirbyteMessageConsumer getSerializedMessageConsumer(final JsonN
     final boolean useAsyncSnowflake = false;
     // final boolean useAsyncSnowflake = config.has("loading_method")
     // && config.get("loading_method").has("method")
-    // && config.get("loading_method").get("method").asText().equals("Internal Staging");
+    // && config.get("loading_method").get("method").asText().equals("Internal Staging")
```

Comment:
is this all supposed to be commented out?

Reply:
Yes, the branch that actually turns on Snowflake is [here](https://github.com/airbytehq/airbyte/pull/26776). That branch will rebase on top of master each time a change to async is made

The rationale originally was that it would be easier once rollout has reached nearly 100% to merge and bump that PR  in one swoop